cmd/kube-controller-manager/app/options/options.go

```diff
@@ -85,6 +85,7 @@ type VolumeConfigFlags struct {
 	PersistentVolumeRecyclerMinimumTimeoutHostPath      int
 	PersistentVolumeRecyclerIncrementTimeoutHostPath    int
 	EnableHostPathProvisioning                          bool
+	PVRecycleRetryCount                                 int
```

Comment:
For consistency, I think this should be `PersistentVolumeRecyclerRetryCount`, which would match the other variables in this struct.